Integrathon documentation improvement (#1896)
SEKOIA-IO/documentation: docs/integration/develop_integration/overview.md

    - Best practices for [Authentications](../formats/best_practices/authentications/) logs
    - An E-learning module for the development of custom format is available in our training catalog, you can request an access [here](https://www.sekoia.io/en/lets-talk-about-your-training-project/)

Once you are done, you will create a custom intake based using this custom format. A custom intake is an instance of your custom format.

1. Click on the `+ Intake` from the Intakes homepage
2. Choose the custom format that you have created (you can search for custom format by its name)
3. Click on the `Create` button of the custom format
4. Provide an intelligible name
5. Select the entity to which you want to associate the corresponding data
6. Click on `Create`
7. Find your newly created source of events as well as the intake key in the Intakes homepage

You have now successfully ingested data from a new product. If you would like to share your custom format to the cybersecurity community or let Sekoia handle the maintenance of this custom format, proceed to the next step.

#### Step 3. Homologate your custom format in the Sekoia catalog (optional)

> You have developed a custom format and would like Sekoia to perform maintenance of this custom format, you can request an homologation. Once homologated, the custom format will integrate the public catalog of Sekoia.

Author: ```suggestion
If you have created a new format and want Sekoia to manage its maintenance, you can request homologation for your custom format. Once approved, the format will be added to Sekoia's public catalog.
```

!!! info
    To homologate your custom format, you will need to contribute through our Github repository 

- Fork the Github repository [SEKOIA-IO/intake-formats](https://github.com/SEKOIA-IO/intake-formats)
- Follow this contribution [guide](../formats/contribute/)

### Usecase 2. You want to modify an existing intake

You would like to parse additional fields in an existing intake or modify the way existing fields are extracted. To do this, you can create a custom format based on an existing intake:

1. Click on the `+ Intake` from the Intakes homepage
2. Search for the intake you wish to modify
3. Click on the `See format` button at the top right of the card
4. Click on the `Duplicate` button at the top right of the custom parser editor
5. Modify the parser by using this [guide](../formats/parser/) 

Once you have finished your modifications, [create a custom intake](#step-2-create-your-custom-intake) based on this new custom format.

### Usecase 3. You want to create a new automation

!!! warning

>     Please note that all automations are available in the public library of Sekoia SOC platform. Other customers of Sekoia can access your automation module. We are still working on the implementation of private automations.

Author: ```suggestion
    Please note that all automations are available in the public library of the Sekoia SOC platform, making them accessible to other Sekoia customers. We are currently working on implementing private automations.
```
